perses/samplers/samplers.py

```diff
@@ -230,6 +230,8 @@ def createContext(self, integrator=None, platform=None):
            If not specified, a VerletIntegrator with 1 fs timestep is created.
         platform : simtk.openmm.Platform, optional, default=None
            If specified, the Platform to use for context creation.
+        thermodynamic_state : ThermodynamicState, optional, default=None
+            If specified, a barostat will be added to periodic systems with a pressure defined.
```

Comment:
I think this part of the docstring is a bit confusing. It's not the case that if `ThermodynamicState` is specified, a barostat will be added, right (that would make no sense if pressure were not specified in `ThermodynamicState`, no?).

 I think the intention is that a barostat will be added if a pressure is specified in `ThermodynamicState`, but since that argument is optional, it's ambiguous. If you agree, I can update this docstring.